plugin/PDFView.qml

```diff
@@ -124,12 +125,36 @@ SilicaFlickable {
     // Ensure proper zooming level when device is rotated.
     onWidthChanged: adjust()
 
+    PDF.Selection {
+        id: selection
+        canvas: pdfCanvas
+        wiggle: Theme.itemSizeSmall / 2
+        onTextChanged: if (text.length > 0) Clipboard.text = text
```

Comment:
Would be more efficient if clipboard is updated only when cursor grab gets released.